### ISSUE TITLE
[Captcha update] all scanner die

### ISSUE THREAD

**Issue by xquiaz** [favll/pogom#413]
## Checklist
- [x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
- [x] I have checked if this issue already exists (both open and closed issues)
- [x] I am using the most recent version of pogom from the master branch
- [ ] If this is an installation issue, I am posting my OS, OS version, python version, pip version
- [ ] I'm posting the complete log
- [ ] I'm using a descriptive title for this issue
## My Issue

after security update needed reCaptcha verification in-game all bot account suddenly death/not working. i wish the dev implemented auto reCaptcha in python build api.
...

**Comment by WillyMaikowski** [comment]
👍  I dont know if pgoapi is working in a method... we must wait or help :P

**Comment by xquiaz** [comment]
after running 1 instance within 2 hours, captcha show up.
![untitled](https://cloud.githubusercontent.com/assets/20725812/19158041/75110f58-8c11-11e6-80dc-e1a6d1c32281.png)

**Comment by Krooked733** [comment]
Same Problem here. Maybe these ingame captchas Appear  only once? Wehre is the trigger?

**Comment by MasterCynder** [comment]
@Krooked : No, the problem don't appear only once. The captcha coul be complete successfull on a smartphone, the captcha appear again few minutes/hours after.

**Comment by Krooked733** [comment]
Game Over?
Right now,Iam testing with 5 Accounts - no Problems so far (2 hours).

**Comment by WillyMaikowski** [comment]
@Krooked733 you always have the option to show the captcha to the user (in the map), but it's very ugly U.u and depends of the users

**Comment by Krooked733** [comment]
@WillyMaikowski 
How is that possible? Is there a way to get/Print the captcha URL at the console/bash?

**Comment by shamik134** [comment]
I still cannot see any pokemon from the scanner.
Hits "received valid response but without any data. ..."
Any fix on that ?
